README.md

```diff
@@ -136,3 +136,37 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+
+When a tag is created in the [`ansible/ansible`](https://github.com/ansible/ansible) repository for a release or release candidate, a corresponding tag should be created in this `ansible-documentation` repository.
+
+First, install the additional tagging dependencies as follows, creating or activating a `venv` as needed:
+
+```bash
+python3 -m venv ./venv
+source ./venv/bin/activated 
+pip -r hacking/tagger/requirements.txt
```

Comment:
```suggestion
pip install -r hacking/tagger/requirements.txt
```

Reply:
Found that as I was walking through it to test the repo detection. Already fixed